src/doc/src/commands/cargo-update.md

```diff
@@ -38,7 +38,7 @@ Cannot be used with <code>--precise</code>.</p>
 </dd>
 
 
-<dt class="option-term" id="option-cargo-update---precise"><a class="option-anchor" href="#option-cargo-update---precise"><code>--precise</code> <em>precise</em></a></dt>
+<dt class="option-term" id="option-cargo-update---precise"><a class="option-anchor" href="#option-cargo-update---precise"><code>--precise</code> <em>version</em></a></dt>
```

Comment:
This isn't just a version.  It can also be a git commit sha

Reply:
I don't feel like `version` is better than `precise`. It implies a version, which people will stop reading and don't know it accept Git revision. At least `precise` is ambiguous enough that people may dig deeper.

> I'm not seeing any precedence for

`--config` has that:

>  --config KEY=VALUE or PATH

Reply:
... and of course the man page is inconsistent on SCREAMING or lower case.